core/domain/state_domain.py

```diff
@@ -828,12 +828,13 @@ def get_all_html_content_strings(self):
         return html_list
 
     @staticmethod
-    def convert_html_in_interaction(interaction_dict, conversion_fn):
+    def convert_html_in_interaction(interaction_dict, ca_specs, conversion_fn):
```

Comment:
Shouldn't the argument be ca_specs_dict?

Reply:
done!